src/main/java/frc/robot/commands/ShootFromLine.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.FalconDriveSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.subsystems.StorageSubsystem;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.ShooterHoodSubsystem;
+
+public class ShootFromLine extends CommandBase {
+  private final FalconDriveSubsystem m_drive;
+  private final ShooterSubsystem m_shooter;
+  private final StorageSubsystem m_storage;
+  private final IntakeSubsystem m_intake;
+
+  public ShootFromLine(FalconDriveSubsystem subsystem, ShooterSubsystem shooter, StorageSubsystem storage,
+      IntakeSubsystem intake, ShooterHoodSubsystem hood) {
+    m_drive = subsystem;
+    m_shooter = shooter;
+    m_storage = storage;
+    m_intake = intake;
+
+    addRequirements(subsystem, shooter, storage, intake, hood);
+  }
+
+  @Override
+  public void initialize() {
+
+  }
+
+  @Override
+  public void execute() {
+    double time = Timer.getMatchTime();
+    m_intake.raiseIntake();
+    if (13 > time && time > 5) {
```

Comment:
The `time > 5` clause is redundant

Reply:
Not on this line.  He's trying to express 13 > time > 5